backend-tests/tests/test_rbac_v2.py

```diff
@@ -36,21 +36,32 @@
 from testutils.api.client import ApiClient
 
 
-def login(user):
+def login(user, use_personal_access_token=False):
     """
     login authenticates the user and saves the user token
     with user object
+    in case of use_personal_access_token===True it saves the personal access token
     """
     useradm_MGMT = ApiClient(useradm.URL_MGMT)
     rsp = useradm_MGMT.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
     assert rsp.status_code == 200, "Failed to setup test environment"
     user.token = rsp.text
 
+    if use_personal_access_token:
+        pat_req = {"name": "my_token_0", "expires_in": 1024}
+        rsp = useradm_MGMT.with_auth(user.token).call(
+            "POST", useradm.URL_TOKENS, pat_req
+        )
+        assert (
+            rsp.status_code == 200
+        ), "Failed to setup test environment with personal token"
+        user.token = rsp.text
```

Comment:
It's not good practice to edit objects fields outside the class - `login` function should be `user` object method IMHO

Reply:
yeah I know, but it was there. do you think I should fix that everywhere? or perhaps in another PR?

Reply:
Another PR; another epic ;-) Resolving since we cannot do about it here.